specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/networkSecurityGroup.json

```diff
@@ -897,6 +897,14 @@
           },
           "description": "A collection of references to subnets."
         },
+        "flowLogs": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "./networkWatcher.json#/definitions/FlowLog"
+          },
+          "description": "A collection of references to flow log resources."
```

Comment:
If it's a collection of references should ref type be `./network.json#/definitions/SubResource`?

Reply:
Resolved offline